src/app/extensions.cpp

```diff
@@ -283,6 +284,8 @@ void Extension::addTheme(const std::string& id,
                          const std::string& path,
                          const std::string& variant)
 {
+  if (id == "default" && !isDefaultTheme())
+    return;
```

Comment:
Add some comment and extra constant (should we add an `ASSERT()` too?):
```suggestion
  // Avoid adding a theme with "default" ID if it's not
  // the default aseprite-theme extension.
  if (id == kAsepriteDefaultThemeName && !isDefaultTheme())
    return;
```
So we have `kAsepriteDefaultThemeExtensionName = "aseprite-theme"` and a new `kAsepriteDefaultThemeName = "default"`.